### ISSUE TITLE
Geeqie 1.5.1 scaling shaking on startup

### ISSUE THREAD

**Issue by Massimo-B** [BestImageViewer/geeqie#826]
### ISSUE TYPE

* Bug Report

### GEEQIE VERSION

```
Geeqie 1.5.1
```

```
x11-libs/gtk+-3.24.22

    Config results:
    -=-=-=-=-=-=-=-=-

Package:
  Name:          geeqie
  Version:       1.5.1

Architecture:
  UNIX:          yes
  Win32:         no (native: no)

Flags:
  Geeqie:        
  DEFS:          -DHAVE_CONFIG_H
  CPPFLAGS:      
  CFLAGS:        -march=native -mtune=native -O2 -pipe -fomit-frame-pointer -Wno-error=deprecated-declarations -Wno-error=sign-compare -Wno-error=return-type
  CXXFLAGS:      -march=native -mtune=native -O2 -pipe -fomit-frame-pointer -fvisibility-inlines-hidden
  Gtk:           -pthread -I/usr/include/gtk-3.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib64/dbus-1.0/include -I/usr/include/gtk-3.0 -I/usr/include/gio-unix-2.0 -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/include/harfbuzz -I/usr/include/pango-1.0 -I/usr/include/fribidi -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/lib64/libffi/include -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include
  Glib:          -pthread -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include
  Thread:        -lgthread-2.0 -pthread -lglib-2.0
  Others:	 -ljpeg -ltiff -llcms2 -lexiv2 -lclutter-1.0 -lcogl-path -latk-1.0 -lcogl-pango -lcogl -lgmodule-2.0 -pthread -lgbm -ldrm -lEGL -lXrandr -ljson-glib-1.0 -lgio-2.0 -lgdk-3 -lpangocairo-1.0 -lpango-1.0 -lgdk_pixbuf-2.0 -lcairo-gobject -lcairo -lgobject-2.0 -lglib-2.0 -lX11 -lXext -lXdamage -lXfixes -lXcomposite -lXi -lclutter-gtk-1.0 -lclutter-1.0 -lcogl-path -lcogl-pango -lcogl -lgmodule-2.0 -pthread -lgbm -ldrm -lEGL -lXrandr -ljson-glib-1.0 -lX11 -lXext -lXdamage -lXfixes -lXcomposite -lXi -lgtk-3 -lgdk-3 -lpangocairo-1.0 -lpango-1.0 -latk-1.0 -lcairo-gobject -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lgobject-2.0 -lglib-2.0 -lchamplain-0.12 -lclutter-1.0 -lcogl-path -latk-1.0 -lcogl-pango -lcogl -lgmodule-2.0 -pthread -lgbm -ldrm -lEGL -lXrandr -ljson-glib-1.0 -lgio-2.0 -lgdk-3 -lpangocairo-1.0 -lpango-1.0 -lgdk_pixbuf-2.0 -lcairo-gobject -lcairo -lgobject-2.0 -lglib-2.0 -lX11 -lXext -lXdamage -lXfixes -lXcomposite -lXi -lchamplain-gtk-0.12 -lclutter-gtk-1.0 -lgtk-3 -lchamplain-0.12 -lclutter-1.0 -lcogl-path -latk-1.0 -lcogl-pango -lcogl -lgmodule-2.0 -pthread -lgbm -ldrm -lEGL -lXrandr -ljson-glib-1.0 -lgio-2.0 -lgdk-3 -lpangocairo-1.0 -lpango-1.0 -lgdk_pixbuf-2.0 -lcairo-gobject -lcairo -lgobject-2.0 -lglib-2.0 -lX11 -lXext -lXdamage -lXfixes -lXcomposite -lXi 

Localization:
  NLS support:   yes
  LINGUAS:        de

Settings:
  Developer:     no
  Debug flags:   no
  Debug log:     no
  Deprecated:    no

Support:
  LCMS:          yes
  Exiv2:	 yes
  Lirc:          no
  Clutter:       yes
  Libchamplain:		yes
  Libchamplain-gtk:	yes
  Lua:	         disabled
  FFmpegthumbnailer:	yes
  Pdf:	         yes

Documentation:
  Doxygen:       /usr/bin/doxygen
  doc-tool:      /usr/bin/gnome-doc-tool


  Now you can type "make" to build Geeqie
  (or you take blue pill and the story ends :)
```

### OS / DISTRIBUTION

Gentoo Linux

### SUMMARY

On Startup, the scaling is shaking until I resize the side bar.
Please take a look at the demo screen record: https://www.youtube.com/watch?v=FYaU-4xn-mA

**Comment by Massimo-B** [comment]
Hello, did you ever reproduce this issue?
Is this related to #848?

**Comment by caclark** [comment]
Fixed in commit 89c5590
